toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -29,6 +29,8 @@ Starts a docker container with the specified version of mariner.
 
```

Comment:
Please add [SPECS_DIR=/path/to/SPECS_DIR/to/use] to Usage on #26

Reply:
Sure :) Done